src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -631,6 +633,16 @@ public void clearAttributes() {
 
     @Override
     public GATKRead copy() {
+        // workaround until https://github.com/google/google-http-java-client/issues/293 is released
+        genomicsRead.setAlignedQuality(new ArrayList<>(genomicsRead.getAlignedQuality()));
+        Map<String, List<String>> infoCopy = new HashMap<>();
+        for (Map.Entry<String, List<String>> entry : genomicsRead.getInfo().entrySet()) {
+            infoCopy.put(entry.getKey(), new ArrayList<>(entry.getValue()));
+        }
+        genomicsRead.setInfo(infoCopy);
+        genomicsRead.getAlignment().setCigar(new ArrayList<>(genomicsRead.getAlignment().getCigar()));
+        // end workaround
```

Comment:
They don't. I tried that in an earlier build [7a2a831](https://github.com/broadinstitute/hellbender/commit/7a2a8319bcec80d19e6a1aeb2cf61f6f831fddbe). Here's the [Travis report](https://travis-ci.org/broadinstitute/hellbender/jobs/76221794). It cuts off the good parts and I didn't keep my own copy, but there was some other type that the reflection-based serialization had trouble with.